stumpy/floss.py

```diff
@@ -486,6 +517,12 @@ def __init__(
         self._finite_T[~np.isfinite(self._finite_T)] = 0.0
         self._finite_Q = self._finite_T[-self._m :].copy()
 
+        if self._normalize:
+            self._T_subseq_isconstant = core.rolling_isconstant(
+                self._T, self._m, self._T_subseq_isconstant_func
+            )
+            self._M_T, self._Σ_T = core.compute_mean_std(self._T, self._m)
```

Comment:
When we close this PR, we should also close #778  as well, right?

Reply:
Yes, we can close that issue.